packages/rest/src/rest-http-error.ts

```diff
@@ -12,7 +13,25 @@ export namespace RestHttpErrors {
     const msg = `Parameters with "in: ${location}" are not supported yet.`;
     return new HttpErrors.NotImplemented(msg);
   }
-  export function invalidRequestBody(msg: string): HttpErrors.HttpError {
-    return new HttpErrors.UnprocessableEntity(msg);
+  export const INVALID_REQUEST_BODY_MESSAGE =
+    'The request body is invalid. See error object `details` property for more info.';
+  export function invalidRequestBody(): HttpErrors.HttpError {
+    return new HttpErrors.UnprocessableEntity(INVALID_REQUEST_BODY_MESSAGE);
   }
+  /**
+   * An invalid request body error contains a `details` property as the machine-readable error.
+   * Each entry in `error.details` contains the following attributes:
```

Comment:
I think we should move documentation of individual properties into tsdoc entries for each property. Thoughts?

```ts
export interface ErrorDetails {
  /** 
   * A path to the invalid field, e.g. `.name`
   */
  path: string;
  
  // ...
};
```